k-distribution/tests/regression-new/markdownSelectors/Makefile

```
@@ -2,7 +2,7 @@ DEF=test
 EXT=test
 TESTDIR=.
 KOMPILE_BACKEND=haskell
-KOMPILE_FLAGS=--md-selector 'k|(keep&!discard)'
+KOMPILE_FLAGS=--md-selector '(k&!discard)|(keep&!discard)'
```

```suggestion
KOMPILE_FLAGS=--md-selector '(k|keep) & !discard'
```
saves a bit of repetition.

---

Why do you need to modify this file?
Just change this
```
{.k .discard .numberLines startFrom="0"}
to this:
{.discard .numberLines startFrom="0"}
```

---

I think the idea is to reproduce precisely the behavior we expect to see in the production code. Although, I don't believe that it's a strong need.

---

But I like @Baltoli's idea! It's even clearer than before while preserving the behavior!

---

I usually prefer minimizing the diff and keeping the PR focused on one thing.
I added that expression to test that we can parse all the elements, not to make any particular sense.
But ok, I guess...